catalyst/dl/callbacks/criterion.py

```diff
@@ -245,8 +252,39 @@ def on_batch_end(self, state: RunnerState) -> None:
         _add_loss_to_state(self.prefix, state, loss)
 
 
+class WeightedCriterionAggregatorCallback(CriterionAggregatorCallback):
```

Comment:
🤔  why don't you use https://github.com/catalyst-team/catalyst/blob/91b514832f77982f83a69494efeb2d8409c7d13d/catalyst/dl/callbacks/criterion.py#L198 ?

Reply:
OK

Reply:
I'm sure there wasn't "weighted_sum" just a week ago, anyway I removed that Weighted additional aggregator.

Reply:
I also made a small unrelated improvement to `CriterionAggregationCallback` - added `weighted_mean` which may be useful for some cases. Hope you will allow that